src/openfga.ne

```diff
@@ -1,11 +1,25 @@
 @preprocessor typescript
 
 types           -> (_newline):* (type (relations (define):+):*):* {%
-    // @ts-ignore
-    data => data[1].map((datum) => {
+    data => {
+         // @ts-ignore
+		const types = data[1].map((datum : any) => {
 		    const relations = datum[1][0] ? datum[1][0][1].flat() : [];
 		    return { ...datum[0], relations }
 		})
+		let schemaVersion = "1.0"
```

Comment:
We don't need this here - maybe in the parser itself, if metadata has any data, set the schema?

Reply:
It doesn't _need_ to come from the parser, but it works in two lines, so 🤷